__test__/api/admins/changePassword.test.js

```diff
@@ -0,0 +1,71 @@
+const app = require("../../../server");
+const supertest = require("supertest");
+const mongoose = require("mongoose");
+const AdminModel = require("../../../models/admins");
+const { getOrgToken } = require("../../../utils/auth/tokenGenerator");
+const { comparePassword } = require("../../../utils/auth/passwordUtils");
+
+const request = supertest(app);
+
+describe("PATCH /v1/admins/:adminId/change-password", () => {
```

Comment:
I think you erred in the url path. It's `/v1/admins/change-password`. Though your code passes, it'll be ideal to make this small fix